_data/sites.json

```
@@ -15018,6 +15042,16 @@
         ]
     },
 
+    {
+        "name": "TradeMe",
+        "url": "https://help.trademe.co.nz/hc/en-us/articles/360007000092#h_01GBNRBS8NG0QNNRWXCV7RVWDS",
+        "difficulty": "hard",
+        "notes": "Click on the link below and ask the staff to delete your account",
```

```suggestion
        "notes": "Click on the link and ask the staff to delete your account",
```
The link actually shows up above the notes, but in any case, better not to mention the position.